src/modules/flash/producer/flash_producer.cpp

```diff
@@ -555,27 +575,35 @@ spl::shared_ptr<core::frame_producer> create_producer(const core::frame_producer
 {
     auto template_host = get_template_host(dependencies.format_desc);
 
-    auto filename = env::template_folder() + L"\\" + template_host.filename;
+    auto filename = env::template_folder() + template_host.filename;
```

Comment:
This will break some configurations that don't end with slash

Reply:
Trailing slash is already enforced here and this was resulting in a double slash:
https://github.com/CasparCG/server/blob/94db7f222d852947eb61ae10b3ac6c9cbbd785be/src/common/env.cpp#L107-L110